the-collab-lab/tcl-79-smart-shopping-list
# Implement design for list page

## Description

This pull request implements the design for the list page as specified in the Figma design. The main changes include:
- The layout of the list page has been updated to align with the Figma design.
- The delete alert has been redesigned for improved clarity and user experience.

## Related Issue

Closes #34 

## Acceptance Criteria

- [x] Add list name to list page to be able to know which list you're currently on.
- [x] Follow the figma design for the list page.

## Type of Changes

Enhancement

## Updates

### Before
![Screen Shot 2024-10-03 at 2 05 18 PM](https://github.com/user-attachments/assets/4e87d600-d008-49a9-9042-f7c8e12f2560)
![Screen Shot 2024-10-06 at 7 44 47 AM](https://github.com/user-attachments/assets/182d2dfe-bf29-4d77-9d7e-e5cc7091c7ab)


### After
![Screen Shot 2024-10-06 at 7 21 01 AM](https://github.com/user-attachments/assets/0b37b917-3442-4136-8423-c855356fc076)

![Screen Shot 2024-10-06 at 7 21 13 AM](https://github.com/user-attachments/assets/ef693553-a879-4562-a4dd-662b5b141531)

## Testing Steps / QA Criteria

- Do a `git pull` and `git checkout hm-list-layout`
- Open the homepage by running `npm start`.
- Navigate to the `list` page.
- You can see the newly implemented design for the list page as specified in the Figma design.
- Click the delete button to view the redesigned delete alert.
